### DISCUSSION TITLE
Long context menu support

### DISCUSSION THREAD

**Discussion by krystofkrticka** [moudey/Shell#111] [General]
Hello, today I installed shell but I'm unable to see all context menu entries because my context menu is too long. So scrolling context menu or something like windows has by default when you context menu won't fit on your screen would be useful.

Some of icons where pushed of to more options section but some disappeared completely. 

| This is how long context menu looks with shell enabled: | This is how long context menu looks when just restored and tweaked by StartAllBack: |
|---|---|
| ![2023-01-17_10-59-12](https://user-images.githubusercontent.com/37749100/212869027-ed1e39eb-190e-4884-bd6c-9ee6749791b1.png) | ![2023-01-17_11-01-19](https://user-images.githubusercontent.com/37749100/212869350-795bdbc3-6f80-4266-85d6-aa9c3481fb40.png) |

**Comment by moudey** [comment]
it's too too too long,

You can shorten it by moving items to a submenu, for example

add this code to `static` section

```
item (find='winamp|git|insync|powerrename|sign and encrypt' menu="more options")
```

Or you can long press in an empty place below the context menu and it will scroll the items.

**Reply by ghost** [reply]
This seems to completely remove those items in multiple items context menu.

For example if you select two files and open context menu, the Power Rename is completely gone, it doesn't appear in more options.

**Reply by moudey** [reply]
> This seems to completely remove those items in multiple items context menu.
> 
> For example if you select two files and open context menu, the Power Rename is completely gone, it doesn't appear in more options.

add `mode="multiple"` property 
```
item (mode="multiple" find='winamp|git|insync|powerrename|sign and encrypt' menu="more options")
```

**Reply by ghost** [reply]
![qWUzXIlT_9lzmhBMexIzGT_8DtpsaqTT7Fo-_6a4W7o-output](https://user-images.githubusercontent.com/123339772/215308827-96f0f3ca-84f5-4dca-914e-9fbc4e2ab83c.gif)

**Comment by krystofkrticka** [comment]
@moudey Thank you! I thought that there would be a way but without updated docs there is not much I can do.
